app/assets/javascripts/admin/products/services/unit_prices.js.coffee

```diff
@@ -0,0 +1,32 @@
+angular.module("admin.products").factory "UnitPrices", (VariantUnitManager, localizeCurrencyFilter) ->
+  class UnitPrices
+    @getDisplayableUnitPrice: (price, scale, unit_type, unit_value, variant_unit_name) ->
```

Comment:
Should this maybe be called `displayableUnitPrice` to be more consistent with the other method names?

Reply:
Done.  :)